students/chylak_malgorzata/lesson_02_flow_control/queen_move.py

```diff
@@ -0,0 +1,9 @@
+x1 = int(input())
+y1 = int(input())
+x2 = int(input())
+y2 = int(input())
+
+if x1 == x2 or y1 == y2:
```

Comment:
Not all of the tests passed on Snakify.You should check as well following condition:
(abs(x1 - x2) == abs(y1 - y2)

so if statement should looks:
if x1 == x2 or y1 == y2 or abs(x1 - x2) == abs(y1 - y2)